hedgehog/src/Hedgehog/Internal/Report.hs

```diff
@@ -715,14 +718,54 @@ ppResult name (Report tests discards result) =
         ppDiscardCount discards <>
         ", passed" <+>
         ppTestCount tests <>
-        "."
+        "." <+>
+        ppClassification classes <+>
+        ppCoverage classes
 
     OK ->
       pure . icon SuccessIcon '✓' . WL.annotate SuccessHeader $
         ppName name <+>
         "passed" <+>
         ppTestCount tests <>
-        "."
+        "." <+>
+        ppClassification classes <+>
+        ppCoverage classes
+
+ppClassification :: Classification -> Doc Markup
+ppClassification (Classification classifiers total) =
+  if Map.null classifiers then
+    mempty
+  else
+    (<>) WL.linebreak $ WL.indent 4 . WL.align . WL.vsep $
+      (\(ClassifierName k, v) -> WL.text $ show (occurrenceRate v total) <> "% " <> k) <$> Map.toList classifiers
+
+occurrenceRate :: Classifier -> Integer -> Double
+occurrenceRate (Classifier _ occurrences) total =
+  let
+    percentage :: Double
+    percentage = fromIntegral occurrences / fromIntegral (total - 2) * 100
+    thousandths :: Integer
+    thousandths = round $ percentage * 10
+  in fromIntegral thousandths / 10
+
+ppCoverage :: Classification -> Doc Markup
+ppCoverage (Classification classifiers total)
+  | null coverageLines = mempty
+  | otherwise =
```

Comment:
Sorry, I only pointed out each style issue once I was hoping you would fix the similar ones in the rest of the file.

This should be an `if` same as above